content/docs/instrumenting/exporters.md

```diff
@@ -89,22 +77,32 @@ hosted outside of the Prometheus GitHub organization.
    
 ### Other monitoring systems
    * [Cloud Foundry Firehose exporter](https://github.com/cloudfoundry-community/firehose_exporter)
-   * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
+   * [Collectd exporter](https://github.com/prometheus/collectd_exporter) (*official*)
+   * [Graphite exporter](https://github.com/prometheus/graphite_exporter) (*official*)
    * [Heka dashboard exporter](https://github.com/docker-infra/heka_exporter)
    * [Heka exporter](https://github.com/imgix/heka_exporter)
    * [Munin exporter](https://github.com/pvdh/munin_exporter)
    * [New Relic exporter](https://github.com/jfindley/newrelic_exporter)
+   * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
+   * [StatsD exporter](https://github.com/prometheus/statsd_exporter) (*official*)
+
 
 ### Miscellaneous
    * [BIG-IP exporter](https://github.com/ExpressenAB/bigip_exporter)
    * [BIND exporter](https://github.com/digitalocean/bind_exporter)
+   * [Blackbox exporter](https://github.com/prometheus/blackbox_exporter) (*official*)
    * [Jenkins exporter](https://github.com/lovoo/jenkins_exporter)
+   * [JMX exporter](https://github.com/prometheus/jmx_exporter) (*official*)
+   * [Memcached exporter](https://github.com/prometheus/memcached_exporter) (*official*)
+   * [Mesos task exporter](https://github.com/prometheus/mesos_exporter) (*official*)
    * [Meteor JS web framework exporter](https://atmospherejs.com/sevki/prometheus-exporter)
    * [Minecraft exporter module](https://github.com/Baughn/PrometheusIntegration)
    * [PowerDNS exporter](https://github.com/janeczku/powerdns_exporter)
    * [rTorrent exporter](https://github.com/mdlayher/rtorrent_exporter)
    * [SMTP/Maildir MDA blackbox prober](https://github.com/cherti/mailexporter)
+   * [SNMP exporter](https://github.com/prometheus/snmp_exporter) (*official*)
```

Comment:
Not sure about SNMP, but could conceivably also be classified under "Other monitoring systems"? What does @brian-brazil say?